lib/ldap.js

```diff
@@ -47,7 +52,12 @@ function initializeConnection () {
       res.once('error', function(err) {
         client.removeAllListeners('end');
         clearTimeout(abort);
-        callback(err);
+        // if there are more than one entry matching the search, the server returns the one entry and a SizeLimitExceededError error
+        if (err.name === 'SizeLimitExceededError') {
```

Comment:
```suggestion
        if (err.name === 'SizeLimitExceededError') {
            client.removeAllListeners('error');
            clearTimeout(abort);
            callback();
        }
```

I would also move this block to the start of the function, as we're not sure what the removeAllListeners code does. We saw an issue today where the connector and auth0 got out of sync and auth0 thought we didn't have it installed properly